client/src/components/WrappedTicket/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link, Route } from 'react-router-dom'
+import './style.css'
+const WrappedTicket = (props) => {
+    console.log(props)
+    return (
+        <div className="wrapped-ticket-container">
+            <div> {props.wrappedTickets ? props.wrappedTickets.map(wrappedticket =>
+                <Route>
```

Comment:
this ```Route``` tag will be repeated for every ticket that arrives.
It should be in the line above the map.

Reply:
you can conditionally render the ```Route``` component so it won't render if there are no wrapped tickets.
```suggestion
                {props.wrappedTickets > 0 && <Route>}
```